app/src/main/res/layout/activity_main.xml

```diff
@@ -32,17 +32,19 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent" />
 
-    <Button
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
```

Comment:
In this file remove the textview that displays "NOTE-IT" at the top, then adjust the position of the listview so that it stays above the addNote button , give it some padding so it looks good.